packages/dscc-gen/package.json

```diff
@@ -47,9 +47,10 @@
   "dependencies": {
     "@google/clasp": "2.2.1",
     "@types/mz": "^0.0.32",
+    "@types/node": "^12.12.6",
```

Comment:
General question. What version of node do we want to support?

Reply:
node 10, as discussed. will address everything else and then refactor towards that.

Reply:
I think I'll refactor to only need node 10, but will not yet merge code between scripts + gen.

Reply:
I *think* the update is complete.